website/docs/plugin/framework-benefits.mdx

```diff
@@ -1,10 +1,12 @@
 ---
-page_title: 'Home - Plugin Development: Plugin Framework Benefits'
+page_title: Plugin framework benefits
 description: >-
-  Compare the prior SDKv2 and the framework for provider development. The framework is easier and more extensible than SDKv2.
+  The plugin framework is an SDK for Terraform provider development that you can
+  use to develop your own Terraform providers. Learn about the benefits of the
+  provider Framework over SDKv2.
```

Comment:
We refer to both the provider and plugin framework in the description. Which should it be?